templates/_helpers.tpl

```diff
@@ -68,7 +68,7 @@ Create the name of the service account to use
 Generate content of spring config for rabbitmq
 */}}
 {{- define "geoserver.spring.rabbitmq" -}}
-host: {{ include "geoserver.fullname" . }}-rabbitmq
+host: {{ .Release.Name }}-rabbitmq
```

Comment:
With that, you can't include tow times the chart in another chart with alias, but it's probably not the goal :-)

Reply:
in the end we should have a dedicated rabbitmq imo